### ISSUE TITLE
Build fails due to `make` built by us

### ISSUE THREAD

**Issue by easybe** [obdev/CrossPack-AVR#10]
At least on Mac OS 10.11+ GNU make version 3.81 or newer is installed, so we do not need to build our own. We could check the installed version but that would require some extra lines. What do you suggest?

Error:

    dyld: lazy symbol binding failed: Symbol not found: _clock_gettime
      Referenced from: /usr/local/CrossPack-AVR-20131216/bin/make
      Expected in: /usr/lib/libSystem.B.dylib

**Comment by starkjohann** [comment]
You can always use the system's make utility, but I could not reproduce this issue. My `make` from CrossPack (same version) does not reference `clock_gettime`.

**Comment by easybe** [comment]
Did you build `make` on Mac OS 10.11+ ? If so, that's strange.

**Comment by starkjohann** [comment]
I've installed the binary from our web site, as far as I can remember. This has been built before 10.11 and I'm running it on 10.12 now. There is no reference to `clock_gettime` in the `make` binary:
```
$ cat /usr/local/CrossPack-AVR-20131216/bin/make | grep 'clock_gettime'
```
(prints no output).

**Comment by easybe** [comment]
Apparently `clock_gettime()` is not available on 10.11 (and probably older), on 10.12 it is. I think what happened was that the binary was linked against the 10.12 SDK and when I tried to run it on 10.11 it couldn't find the symbol. However, let's just use system's `make`.
